core/authority-discovery/src/lib.rs

```diff
@@ -244,22 +260,18 @@ where
 			let schema::SignedAuthorityAddresses {
 				signature,
 				addresses,
-			} = schema::SignedAuthorityAddresses::decode(value).map_err(Error::Decoding)?;
-			let signature = Signature(signature);
-
-			let is_verified = self
-				.client
-				.runtime_api()
-				.verify(&id, &addresses, &signature, &authority_id.clone())
-				.map_err(Error::CallingRuntime)?;
+			} = schema::SignedAuthorityAddresses::decode(value).map_err(Error::DecodingProto)?;
+			// TODO: Should we rather have a `TryFrom<&[u8]> for Signature` instead of encoding a scale encoded
+			// signature as Protobuf?
+			let signature = AuthoritySignature::decode(&mut &signature[..]).map_err(Error::EncodingDecodingScale)?;
```

Comment:
Why not use SCALE here?

Reply:
Friendly ping @bkchr.

Reply:
Sounds reasonable with the `[u8]`.

Reply:
After a private conversation with @bkchr, we settled on first SCALE and then Protobuf encoding as it is implemented right now. Feel free to revive the discussion.